From 9948e392ec0ad9198d10db55d3302e2f31a915cb Mon Sep 17 00:00:00 2001
Message-ID: <9948e392ec0ad9198d10db55d3302e2f31a915cb.1732837202.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1732837202.git.sharlatanus@gmail.com>
References: <cover.1732837202.git.sharlatanus@gmail.com>
From: "Artyom V. Poptsov" <poptsov.artyom@gmail.com>
Date: Sun, 24 Nov 2024 13:27:35 +0300
Subject: [PATCH 11/15] gnu: Add go-github-com-muesli-ansi.

* gnu/packages/golang-xyz.scm (go-github-com-muesli-ansi): New variable.

Change-Id: I3dbbd8af04373997fcb3c60872dff14ab1b2b76a
Reviewed-by: Sharlatan Hellseher <sharlatanus@gmail.com>
---
 gnu/packages/golang-xyz.scm | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index 6d54c9efbd..a9ab207c55 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -5731,6 +5731,31 @@ (define-public go-github-com-msteinert-pam-v2
        ((#:import-path _ "github.com/msteinert/pam")
         "github.com/msteinert/pam/v2")))))
 
+(define-public go-github-com-muesli-ansi
+  (package
+    (name "go-github-com-muesli-ansi")
+    (version "0.0.0-20230316100256-276c6243b2f6")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/muesli/ansi")
+             (commit (go-version->git-ref version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1jr8kgn3vb72jmf4a8n52g876mfpbvk3310p8gsg7jkn338af4m9"))))
+    (build-system go-build-system)
+    (arguments
+     (list #:import-path "github.com/muesli/ansi"))
+    (propagated-inputs
+     (list go-github-com-mattn-go-runewidth
+           go-github-com-rivo-uniseg))
+    (home-page "https://github.com/muesli/ansi")
+    (synopsis "Raw ANSI sequence helpers")
+    (description
+     "ANSI sequence helpers for working with raw ANSI sequences.")
+    (license license:expat)))
+
 (define-public go-github-com-muesli-cancelreader
   (package
     (name "go-github-com-muesli-cancelreader")
-- 
2.46.0

